src/js/components/deployments/deployment-report/devicelist.js

```diff
@@ -79,11 +84,13 @@ const ProgressDeviceList = ({ created, devices, globalSettings, viewLog }) => {
             <TableCell style={headerStyle} tooltip={globalSettings.id_attribute || 'Device ID'}>
               {globalSettings.id_attribute || 'Device ID'}
             </TableCell>
-            {['Device type', 'Current software', 'Started', 'Finished', 'Deployment status', ''].map((content, index) => (
-              <TableCell key={`device-list-header-${index + 1}`} style={headerStyle} tooltip={content}>
-                {content}
-              </TableCell>
-            ))}
+            {['Device type', 'Current software', 'Started', 'Finished', 'Retries', 'Deployment status', ''].map((content, index) =>
+              content != 'Retries' || retries ? (
+                <TableCell key={`device-list-header-${index + 1}`} style={headerStyle} tooltip={content}>
+                  {content}
+                </TableCell>
+              ) : null
+            )}
```

Comment:
```suggestion
            {['Device type', 'Current software', 'Started', 'Finished', '', 'Deployment status', ''].map((content, index) => (
                <TableCell key={`device-list-header-${index + 1}`} style={headerStyle} tooltip={content}>
                  {content}
                </TableCell>
            ))}
```
might be enough

Reply:
@mzedel with this change, we lose the Retries header, don't we?
We end up with a 1/3 table cell but we don't say what it means.
I'm not sure I get your point.

Reply:
argh right, I misread the condition and just focused on the `content != 'Retries'` - sorry about that!